appservice/closed-session.html

```diff
@@ -3,10 +3,38 @@
 <head>
     <title>Web Console</title>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="patternfly.css">
 </head>
-<body>
-    <h1>Web Console</h1>
-
-    <p>The Web Console session ended. You can close this window now.</p>
+<body class="pf-m-redhat-updated-font pf-m-redhatmono-font">
+  <div id="root">
+    <div tabindex="-1" id="ws-router" style="outline: none;">
+      <div id="ws-html-demos-c-page-basic" class=
```

Comment:
"ws-html-demos-" sounds like a copy&paste leftover? Same for the "ws-router"?

Reply:
All dropped.